server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -1043,6 +1043,11 @@ public void verify(String seed, DiscoveryNode localNode) {
                     throw new RepositoryVerificationException(metadata.name(), "store location [" + blobStore() +
                         "] is not accessible on the node [" + localNode + "]", exp);
                 }
+                final String seedRead = Streams.readFully(masterDat).utf8ToString();
```

Comment:
Probably move this check up? This would be easier to read - we read something, we verify something and move to something different.

Reply:
Yea for sure cleaning that up in a sec ... this thing was weird from the get go, thanks for spotting :)